fbchat/graphql.py

```diff
@@ -193,6 +193,24 @@ def graphql_to_plan(a):
     rtn.invited = [m.get('node').get('id') for m in guests if m.get('guest_list_state') == "INVITED"]
     return rtn
 
+def graphql_to_quick_reply(q, is_response=False):
+    data = dict()
+    if q.get('title') is not None: data["title"] = q["title"]
+    if q.get('image_url') is not None: data["image_url"] = q["image_url"]
+    if q.get('payload') is not None: data["payload"] = q["payload"]
+    if q.get('data') is not None: data["data"] = q["data"]
```

Comment:
Getting the `payload` and `data` attributes outside like this is fine, since they're attributes on the `QuickReply` model, but `title` is only relevant on `QuickReplyText`, so it should only be added there